w_common/lib/src/common/disposable.dart

```diff
@@ -365,7 +365,7 @@ class Disposable implements _Disposable, DisposableManagerV7, LeakFlagger {
   /// Dispose of the object, cleaning up to prevent memory leaks.
   @override
   Future<Null> dispose() async {
-    Stopwatch stopwatch;
+    late Stopwatch stopwatch;
```

Comment:
Since we conditionally initialize `stopwatch`, I think it would be better to make stopwatch nullable and use `!` when we are confident it is not `null`.